tests/test_strain_design_deterministic.py

```diff
@@ -146,3 +160,31 @@ def test_invalid_input(self, cplex_optknock):
             optknock.run(target="EX_ac_lp_e_rp_")
         with pytest.raises(ValueError):
             optknock.run(biomass="Biomass_Ecoli_core_N_lp_w_fsh_GAM_rp__Nmet2")
+
+
+@pytest.mark.skipif('gurobi' not in solvers, reason="No gurobi interface available")
+class TestGrowthCouplingPotential:
+    def test_growth_coupling_potential_runs(self, gurobi_growth_coupling_potential):
+        _, growth_coupling_potential = gurobi_growth_coupling_potential
+        result = growth_coupling_potential.run()
+        assert "obj_val" in result
```

Comment:
This seems like it's not a very meaningful test. As long as there is no exception in the `run` method, these keys will always be present.